definitions/npm/commander_v2.x.x/flow_>=0.28.x/commander_v2.x.x.js

```diff
@@ -0,0 +1,276 @@
+   *        .action(function(){
+   *          console.log('setup');
+   *        });
+   *
+   *      program
+   *        .command('exec <cmd>')
+   *        .description('run the given remote command')
+   *        .action(function(cmd){
+   *          console.log('exec "%s"', cmd);
+   *        });
+   *
+   *      program
+   *        .command('*')
+   *        .description('deploy the given env')
+   *        .action(function(env){
+   *          console.log('deploying "%s"', env);
+   *        });
+   *
+   *      program.parse(process.argv);
+   *
+   * @param {String} name
+   * @param {String} [desc]
+   * @param {Mixed} [opts]
+   * @return {Command} the new command
+   * @api public
+   */
+  command(name: string, desc?: string, opts?: any): $npm$commander$Command;
+
+  /**
+   * Parse expected `args`.
+   *
+   * For example `["[type]"]` becomes `[{ required: false, name: 'type' }]`.
+   *
+   * @param {Array} args
+   * @return {Command} for chaining
+   * @api public
+   */
+  parseExpectedArgs(args: Array<string>): $npm$commander$Command;
+
+  /**
+   * Register callback `fn` for the command.
+   *
+   * Examples:
+   *
+   *      program
+   *        .command('help')
+   *        .description('display verbose help')
+   *        .action(function(){
+   *           // output help here
+   *        });
+   *
+   * @param {Function} fn
+   * @return {Command} for chaining
+   * @api public
+   */
+  action(fn: (...args: any[]) => void): $npm$commander$Command;
+
+  /**
+   * Define option with `flags`, `description` and optional
+   * coercion `fn`.
+   *
+   * The `flags` string should contain both the short and long flags,
+   * separated by comma, a pipe or space. The following are all valid
+   * all will output this way when `--help` is used.
+   *
+   *    "-p, --pepper"
+   *    "-p|--pepper"
+   *    "-p --pepper"
+   *
+   * Examples:
+   *
+   *     // simple boolean defaulting to false
+   *     program.option('-p, --pepper', 'add pepper');
+   *
+   *     --pepper
+   *     program.pepper
+   *     // => Boolean
+   *
+   *     // simple boolean defaulting to true
+   *     program.option('-C, --no-cheese', 'remove cheese');
+   *
+   *     program.cheese
+   *     // => true
+   *
+   *     --no-cheese
+   *     program.cheese
+   *     // => false
+   *
+   *     // required argument
+   *     program.option('-C, --chdir <path>', 'change the working directory');
+   *
+   *     --chdir /tmp
+   *     program.chdir
+   *     // => "/tmp"
+   *
+   *     // optional argument
+   *     program.option('-c, --cheese [type]', 'add cheese [marble]');
+   *
+   * @param {String} flags
+   * @param {String} description
+   * @param {Function|Mixed} fn or default
+   * @param {Mixed} defaultValue
+   * @return {Command} for chaining
+   * @api public
+   */
+  option(flags: string, description?: string, fn?: ((arg1: any, arg2: any) => void) | RegExp, defaultValue?: any): $npm$commander$Command;
```

Comment:
Let's [use `mixed` instead of `any`](https://github.com/flowtype/flow-typed/blob/master/CONTRIBUTING.md#avoid-any-when-possible) for the `defaultValue` parameter